src/tup/parser.c

```diff
@@ -2732,9 +2739,11 @@ static int nl_add_path(struct tupfile *tf, struct path_list *pl,
 		struct variant *variant;
 
 		if(tup_db_select_tent_part(pl->dt, pl->pel->path, pl->pel->len, &tent) < 0) {
-			return -1;
+			return /*args.excluding ? 0 :*/ -1;
```

Comment:
Did you still need to do something with args.excluding here? I'm not sure why this is commented out.